packages/types/uiComponents.ts

```diff
@@ -167,3 +173,14 @@ export interface RevealFormat {
   regex: RegExp;
   replace: string;
 }
+
+export interface FormPayload {
+  value: string;
+  isComplete: boolean;
+}
+
+export interface FormFrameClientMessages extends EvervaultFrameClientMessages {
+  EV_CHANGE: FormPayload;
```

Comment:
We don't seem to ever fire this event. Can we remove it?

Reply:
For sure